### ISSUE TITLE
Newlines in "eval" break "del()" behaviour

### ISSUE THREAD

**Issue by rbo** [mikefarah/yq#855]
**Describe the bug**

If I add newlines into the eval part, for better readablity of scripts, it breaks the dell behavior. It doesn't delete the last element in the list. If I write the same del statement without newlines it works like charm.

Version of yq: 4.9.5
Operating system: linux
Installed via: curl / download from release page

**Input Yaml**
example.yaml:
```yaml
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRoleBinding
metadata:
  annotations:
    include.release.openshift.io/self-managed-h

igh-availability: "true"
    include.release.openshift.io/single-node-developer: "true"
  creationTimestamp: "2021-03-19T18:28:38Z"
  name: machine-api-operator
  resourceVersion: "2156"
  selfLink: /apis/rbac.authorization.k8s.io/v1/clusterrolebindings/machine-api-operator
  uid: 1594b3cb-bfc6-451b-8c32-4e114196b334```
```

**Command**
The command you ran:
```bash
yq eval '
    del(
        .metadata.uid,
        .metadata.creationTimestamp,
        .metadata.managedFields,
        .metadata.resourceVersion,
        .metadata.selfLink
    ) ' example.yaml

```

**Actual behavior**

```yaml
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRoleBinding
metadata:
  annotations:
    include.release.openshift.io/self-managed-high-availability: "true"
    include.release.openshift.io/single-node-developer: "true"
  name: machine-api-operator
  selfLink: /apis/rbac.authorization.k8s.io/v1/clusterrolebindings/machine-api-operator
```

**Expected behavior**

```yaml
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRoleBinding
metadata:
  annotations:
    include.release.openshift.io/self-managed-high-availability: "true"
    include.release.openshift.io/single-node-developer: "true"
  name: machine-api-operator
```

**Additional context**

It doesn't delete `metadata.selfLink`.

Command without new lines works very well
```
yq eval ' del( .metadata.uid, .metadata.creationTimestamp, .metadata.managedFields, .metadata.resourceVersion, .metadata.selfLink) ' example.yaml
```

**My test-case.sh**

<https://gist.github.com/rbo/863e8cfbbb527dbeedc1562a3ef540ba>

```yaml
./test-case.sh
+ cat example.yaml
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRoleBinding
metadata:
  annotations:
    include.release.openshift.io/self-managed-high-availability: "true"
    include.release.openshift.io/single-node-developer: "true"
  creationTimestamp: "2021-03-19T18:28:38Z"
  name: machine-api-operator
  resourceVersion: "2156"
  selfLink: /apis/rbac.authorization.k8s.io/v1/clusterrolebindings/machine-api-operator
  uid: 1594b3cb-bfc6-451b-8c32-4e114196b334
+ echo 'Single line eval pass:'
Single line eval pass:
+ yq eval ' del( .metadata.uid, .metadata.creationTimestamp, .metadata.managedFields, .metadata.resourceVersion, .metadata.selfLink) ' example.yaml
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRoleBinding
metadata:
  annotations:
    include.release.openshift.io/self-managed-high-availability: "true"
    include.release.openshift.io/single-node-developer: "true"
  name: machine-api-operator
+ echo 'Multihttps://docs.google.com/document/d/1yBdx4O1IFtDhKQxAipMr3JC0wbRm98nB31XfI7LE4hg/edit#heading=h.3qjsun3cm2c1 line eval fail:'
Multihttps://docs.google.com/document/d/1yBdx4O1IFtDhKQxAipMr3JC0wbRm98nB31XfI7LE4hg/edit#heading=h.3qjsun3cm2c1 line eval fail:
+ yq eval '
    del(
        .metadata.uid,
        .metadata.creationTimestamp,
        .metadata.managedFields,
        .metadata.resourceVersion,
        .metadata.selfLink
    ) ' example.yaml
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRoleBinding
metadata:
  annotations:
    include.release.openshift.io/self-managed-high-availability: "true"
    include.release.openshift.io/single-node-developer: "true"
  name: machine-api-operator
  selfLink: /apis/rbac.authorization.k8s.io/v1/clusterrolebindings/machine-api-operator
+ yq -V
yq version 4.9.5
```

**Comment by mikefarah** [comment]
Yep it's a bug - will fix

**Comment by mikefarah** [comment]
Fixed in 4.9.6
